17-react-native/classworks/Malkov_Aleksei/screens/Details/Details.screen.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Text, View, Image } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import { FlatList, TouchableHighlight } from 'react-native-gesture-handler';
+import { useRoute } from '@react-navigation/native';
+import { AntDesign } from '@expo/vector-icons';
+import * as actions from '../../resources/actions';
+import * as selectors from '../../resources/selector';
+import styles from './Details.styles';
+
+
+function DetailsScreen() {
+  const dispatch = useDispatch();
+  const { params } = useRoute();
+  const { item } = params;
+  const comics = useSelector(selectors.getMarvelComics);
+  const favourites = useSelector(selectors.getFavMarvels);
+  const [isFavorite, setIsFavorite] = useState(false);
+
+  useEffect(() => {
+    if (favourites.find((favM) => favM.id === item.id)) setIsFavorite(true);
+    else setIsFavorite(false);
+    return () => setIsFavorite(false);
+  }, [favourites]);
+
+  useEffect(() => {
+    dispatch({ type: 'currMarvelId:set', payload: { currMarvelId: item.id } });
+    dispatch(actions.getComicsForMarvel(item.id));
+  }, []);
+
+  function unfavourate(id) {
```

Comment:
why does function named `unfavourate` calls `unfavourite`. the naming of the same parts of the app should be the same.